client-web/src/exlibris-app/exlibris-book-tile.html

```diff
@@ -0,0 +1,212 @@
+  ~ LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+  ~ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+  ~ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  ~ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+  ~ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  -->
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../../bower_components/iron-image/iron-image.html">
+<link rel="import" href="../../bower_components/iron-scroll-threshold/iron-scroll-threshold.html">
+<link rel="import" href="../../bower_components/iron-list/iron-list.html">
+<link rel="import" href="../../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="./book_values.html">
+
+<dom-module id="exlibris-book-tile">
+    <template>
+        <style>
+            :host {
+                display: block;
+            }
+
+            .book-content {
+                @apply --layout-vertical;
+                border-radius: 3px;
+                box-shadow: 4px 4px 10px #888888;
+                cursor: pointer;
+                background-color: black;
+                width: 160px;
+                height: 300px;
+                margin: 20px;
+            }
+
+            iron-image {
+                border-radius: 3px 3px 0 0;
+                height: 75%;
+            }
+
+            iron-image:hover {
+                opacity: 0.5;
+            }
+
+            #synopsis {
+                @apply --layout-vertical;
+                height: 25%;
+                bottom: 0;
+                left: 0;
+                right: 0;
+                background: white;
+            }
+
+            #title {
+                text-overflow: ellipsis;
+                overflow: hidden;
+                width: 145px;
+                white-space: nowrap;
+                color: #212121;
+                font-size: 14px;
+                padding-left: 9px;
+                margin-top: 3px;
+
+            }
+
+            #author {
+                text-overflow: ellipsis;
+                overflow: hidden;
+                width: 150px;
+                white-space: nowrap;
+                color: #616161;
+                font-size: 12px;
+                padding-left: 9px;
+            }
+
+            #status-text {
+                background: var(--exlibris-app-primary-color);
+                color: white;
+                font-size: 12px;
+                text-align: center;
+                margin-top: 5px;
+            }
+
+            #status-text.warn {
+                background: var(--paper-red-400);
+            }
+
+            #status-text.active {
+                background: var(--paper-green-400);
+            }
+        </style>
+
+        <template is="dom-if" if="[[book]]">
```

Comment:
Is it a valid case when book is null?

Reply:
@dkalinin  When there are not enough tiles for one line the iron grid will add empty tile and you will see black tiles. That it's why I have added this `dom-if` and in this case,  this will also be a tile but without styles.
And YES, with `null` works just like with `undefined`.

Reply:
It adds empty tile with `hidden` attribute so it should not be displayed. Adjust your styles.